src/metrics.rs

```diff
@@ -25,7 +25,9 @@ use crate::config::GooseDefaults;
 use crate::goose::{get_base_url, GooseMethod, GooseTaskSet};
 use crate::logger::GooseLog;
 use crate::report;
-use crate::util;
+use crate::util::{
+    median, ms_timer_expired, standard_deviation, timer_expired, truncate_string, MovingAverage,
+};
```

Comment:
This change adds a lot of noise to the diff, and is an unnecessary style change. I'd much prefer to discuss and if we decide to rework the coding style then do it consistently across the entire codebase and not just in this file, which is now inconsistent with the rest of the codebase.

Reply:
Done.